package.json

```diff
@@ -62,7 +62,7 @@
     "next-images": "1.3.0",
     "next-mdx-enhanced": "^2.4.0",
     "next-plugin-yaml": "^1.0.1",
-    "prettier": "^1.19.1",
+    "prettier": "2.0.5",
```

Comment:
There was an issue with the version of prettier that we were using with using next's `Link` component in MDX files: https://github.com/prettier/prettier/issues/7176